app/server/utils/metrics/transfer_stats.py

```diff
@@ -19,30 +19,48 @@ def __init__(self, group_strategy, timeseries_unit = 'day', token=None):
         self.timeseries_unit = timeseries_unit
         self.metrics = []
 
-        total_distributed_query = db.session.query(func.sum(CreditTransfer.transfer_amount).label('total'))
+        total_amount_query = db.session.query(func.sum(CreditTransfer.transfer_amount).label('total'))
         self.metrics.append(metric.Metric(
             metric_name='total_distributed',
-            query=total_distributed_query,
+            query=total_amount_query,
             object_model=CreditTransfer,
             stock_filters=[filters.disbursement_filters],
             caching_combinatory_strategy=metrics_cache.SUM,
             filterable_by=self.filterable_attributes,
             bypass_user_filters=True,
         ))
```

Comment:
If it's the total amount that should be returned, shouldn't the stock filter `filters.disbursement_filters` be removed? Sorry haven't been involved in backend metrics so trying to wrap my head around queries

Reply:
oh `total_amount_query` is the base query that all metrics of this type use. So u can see it's reused agian below. (If u look at the lines removed, they were all just copy pastes, so this cleans up)

```
        total_distributed_query = db.session.query(func.sum(CreditTransfer.transfer_amount).label('total'))
        total_spent_query       = db.session.query(func.sum(CreditTransfer.transfer_amount).label('total'))
        total_exchanged_query   = db.session.query(func.sum(CreditTransfer.transfer_amount).label('total'))
```